sio-digital-team/active-safe
# Risk factors

https://trello.com/c/7sq8mjkI


**Changes:**

- [x] Added risk factor accordions
- [x] Added custom styling to accordions
- [x] Updated Accordion JS to add additional functionality to accordions
  

**How to test:**

- Open an accordion and content is shown
- Open a different accordion item and the previous one should close
- Rapid clicking will not mess up the accordions or "queue" the open and close animation
- Resize browser and text is still legible at all view ports
  
**Hotfixes:**
- [x] Link styling
- [x] Wiggling/resizing container
